packages/bpk-scrim-utils/src/withScrimmedPortal.tsx

```diff
@@ -17,56 +17,51 @@
  */
 
 import type { ComponentType} from 'react';
-import { useEffect, useRef } from 'react';
+import { useState, useEffect, useRef } from 'react';
 import { createPortal } from 'react-dom';
 
 import withScrim from './withScrim';
 import type { Props as ScrimProps } from './withScrim';
 
 export type Props = ScrimProps & {
     renderTarget?: (() => HTMLElement | null) | null;
+    isPortalReady?: boolean;
 };
 
 const getPortalElement = (target: (() => HTMLElement | null) | null | undefined) => {
     const portalElement = target && typeof target === 'function' ? target() : null;
-    
+
     if (portalElement) {
         return portalElement;
     }
-    
+
     if (document.body) {
         return document.body;
     }
     throw new Error('Render target and fallback unavailable');
 }
 
-const withScrimmedPortal = (WrappedComponent: ComponentType<ScrimProps>) => {
+const withScrimmedPortal = (WrappedComponent: ComponentType<Props>) => {
```

Comment:
not sure about this change, is correct? if not TS was complaining

Reply:
Looks right to me :)

Reply:
Hmmmm 🤔 I think `ScrimProps` should've been ok as `Props` just has `renderTarget` in addition, but the wrapped component won't accept `renderTarget`

Reply:
true, changed, it doesn't seem to complain now